datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/asn1/Asn1Encoder.kt

```diff
@@ -160,18 +160,19 @@ class Asn1TreeBuilder {
         elements += value.encodeToAsn1GeneralizedTime()
```

Comment:
Somehow I can't make a comment above ...

Line 137: `Asn1String.Printable` may throw an exception, so this method here `printableString` should be annotated too.